src/envoy/http/mixer/filter.h

```diff
@@ -33,6 +33,30 @@ struct PerRouteServiceConfig : public Router::RouteSpecificFilterConfig {
   std::string hash;
 };
 
+// The struct to store gRPC message counter state.
+struct GrpcMessageCounter {
+  GrpcMessageCounter() : state(ExpectByte0), current_size(0), count(0){};
+
+  // gRPC uses 5 byte header to encode subsequent message length
```

Comment:
Could you add the link to gRPC spec https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md, or state here that gRPC message has 1 byte Compressed-Flag+4 byte Message-Length?

Reply:
The link is in .cc file. It seems that it's more relevant to the implementation than the header.